app/src/androidTest/java/ch/epfl/sdp/peakar/general/SettingsActivityTest.java

```diff
@@ -279,6 +279,59 @@ public void TestAllowCachingButton(){
         assertThat(startVal, not(is(endVal)));
     }
 
+    /*Test preference toggle developer options*/
+    @Test
+    public void TestEnableDeveloperOptions(){
+        SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences(context);
+
+        //Get selected value
+        boolean startVal = sharedPreferences.getBoolean("developer_options", false);
```

Comment:
Use constant string here, as well as in line 292.
Constants can be added to the file **TestingConstants.java**